pkg/providers/amifamily/al2.go

```diff
@@ -41,6 +41,7 @@ type AL2 struct {
 
 func (a AL2) DescribeImageQuery(ctx context.Context, ssmProvider ssm.Provider, k8sVersion string, amiVersion string) (DescribeImageQuery, error) {
 	ids := map[string][]Variant{}
+	var errs []error
```

Comment:
For the other files as well - I don't think we need to aggregate the errors if we ultimately only send the first one back.

Reply:
I changed it to return only the last error.